data/raw/USB6014_core_nodes.txt

```diff
@@ -0,0 +1,14 @@
+51
```

Comment:
Comment for this script either in docs or somewhere someone can interpret what it means